client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx

```diff
@@ -22,6 +22,12 @@ const StyledCurrencyInput = styled(RCInput)(({ theme }) => ({
   border: 'none',
   '&:focus': {
     outline: 'none',
+    borderBottom: '2px solid #FF0000',
+    borderRadius: '8px 8px 0px 0px',
+  },
+  '&:hover': {
+    borderBottom: '2px solid #4F4F4F',
+    borderRadius: '8px 8px 0px 0px',
```

Comment:
The red line is gone now when the mouse is over the control, this is different to the other controls. Could you please have a look?

Reply:
Sorry, I'm not sure what you mean. Which control is this happening on? 

This section is for the currency input
![Screen Shot 2022-11-30 at 2 19 58 PM](https://user-images.githubusercontent.com/61820074/204684245-5fa3aaef-1798-49e5-beb3-e81e1f1e5b91.png)
Red line for focus, and when you hover over it's a grey line like the other input/select components